ui/templates/background-images.css

```diff
@@ -9,10 +9,10 @@
   background-image: url({% static "images/mit_logo_on.svg" %});
 }
 .navbar .navbar-header .micromasters-logo {
-  background-image: url({% static "images/micromasters_logo.svg" %});
+  background-image: url({% static "images/micromasters_logo_programs.svg" %});
 }
 .navbar .navbar-header .micromasters-logo:hover {
-  background-image: url({% static "images/micromasters_logo_on.svg" %});
+  background-image: url({% static "images/micromasters_logo_on_programs.svg" %});
```

Comment:
I don't think we need a special version just for this page. We should change the "logo" everywhere

Reply:
This particular logo is used only in this header on the homepage.

Reply:
Hmm. I have no idea why we do that, but I don't think I want to open than can of worms.